docs/software/mavlink.md

```diff
@@ -41,6 +41,18 @@ ExpressLRS now has full bi-directional [MAVLink](https://mavlink.io/en/) support
 
 3. Select the COM port on the GCS, and connect using `460800` baud
 
+## Using MAVLink on PX4
+1. Configure the UART with SER_TEL2_BAUD to `460800 8N1` (If you are using QGroundControl for that it is under the Parameters -> Serial settings)
+2. Configure MavLink with MAV_0_CONFIG to `TELEM2`
```

Comment:
2. Configure MAVLink with MAV_0_CONFIG to `TELEMx` (where x is the serial port number from the previous point)